{"name": "test_new_order_appears_in_progress_section[firefox]", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'setup' for <Function test_open_order_details_popup[firefox]>>\n\n    @pytest.fixture(params=[\"chrome\", \"firefox\"], scope=\"class\")\n    def setup(request):\n>       driver = initialize_driver(request.param)\n\nconftest.py:6: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nbrowser_name = 'firefox'\n\n    def initialize_driver(browser_name):\n        if browser_name == \"chrome\":\n            driver = webdriver.Chrome(ChromeDriverManager().install())\n        elif browser_name == \"firefox\":\n>           driver = webdriver.Firefox(executable_path=GeckoDriverManager().install())\nE           TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nutils\\driver_setup.py:9: TypeError"}, "parameters": [{"name": "setup", "value": "'firefox'"}], "start": 1730625013531, "stop": 1730625013531, "uuid": "4ac36534-4efb-4661-821a-5b9943b18381", "historyId": "92717638a5254308693d4b011f0ab378", "testCaseId": "be3e01f01148737b669a67bdcd7e0065", "fullName": "tests.test_order_feed.TestOrderFeed#test_new_order_appears_in_progress_section", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "9892-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}